app/scienceapi/scienceblog/views.py

```diff
@@ -0,0 +1,98 @@
+from django.shortcuts import redirect
+from rest_framework.generics import ListAPIView, RetrieveAPIView
+
+from mezzanine.blog.models import BlogPost, BlogCategory
+
+from .serializers import BlogPostSerializer, BlogCategorySerializer
+
+
+def blog_preview(request, *args, **kwargs):
+    # Redirect to preview url on frontend. Using a dummy url for now.
+    return redirect('https://google.com')
```

Comment:
Why do you have this?

Reply:
This is the link used inside preview button. We will change this once changes in frontend lands